packages/rum-react/karma.conf.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2017-present, Elasticsearch BV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+const { baseConfig, prepareConfig } = require('../../dev-utils/karma.js')
+const { getGlobalConfig } = require('../../dev-utils/test-config')
+const { EnvironmentPlugin } = require('webpack')
+const { getWebpackEnv } = require('../../dev-utils/test-config')
+
+module.exports = function(config) {
+  config.set(baseConfig)
+  config.webpack = {
+    devtool: 'source-map',
+    mode: 'development',
+    module: {
+      rules: [
+        {
+          test: /.jsx?$/,
+          exclude: /node_modules/,
+          use: {
+            loader: 'babel-loader',
+            options: {
```

Comment:
you can pass configFile and pass the same one for rum-react/babel.config.js.
Example https://github.com/elastic/apm-agent-rum-js/blob/0ca1e5f97fd4204d6e93d099a9e10b01217bf920/dev-utils/karma.js#L70

Reply:
Let's discuss this in #308, I'm still not sure how much benefit we will gain from this. Specially since in each of the babel files we need to change the config.